README.rst

```diff
@@ -2,15 +2,17 @@
 21cmSense
 =========
 
-.. image:: https://travis-ci.org/steven-murray/21cmSense.svg?branch=master
-    :target: https://travis-ci.org/steven-murray/21cmSense
-.. image:: https://codecov.io/gh/steven-murray/21cmSense/branch/master/graph/badge.svg
-  :target: https://codecov.io/gh/steven-murray/21cmSense
+.. image:: https://img.shields.io/badge/Python-3.9-3776AB.svg?style=flat&logo=python&logoColor=white
+    :target: https://www.python.org
```

Comment:
is this badge for the minimum supported python version? It's been a while since I've done python badge-ing, is there test that lets the user know it is the minimum?

Reply:
Actually I forgot to update the version from the code I snipped off some website. I don't know if it's possible to specify the versions in the badge. Let me have a check

Reply:
Ah, OK I've udpated to what it should be -- which displays all python versions that it supports. It'll only work once we make a pip package for it though

Reply:
wonderful! thank you!